---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER |Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: ec8fa09f34b27d9638f505981f11de960cb4d5b9
ms.sourcegitcommit: f7fced330b64d6616aeb8766747295807c92dd41
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 04/23/2019
ms.locfileid: "62520162"
---
# <a name="localdberrorcannotaccessinstancefolder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|261|  
|イベント ソース|SQL Server Local Database Runtime 12.0|  
|コンポーネント|Local Database Runtime API|  
|メッセージ テキスト|ローカル データベース インスタンスのフォルダーにアクセスできません: %%LOCALAPPDATA%%\Microsoft\Microsoft SQL Server のローカル DB\Instances\\< インスタンス名\>します。|  
  
## <a name="explanation"></a>説明  
 インスタンス フォルダーにアクセスできません。  
  
## <a name="user-action"></a>ユーザーの操作  
  
